help-docs/marketplace/apps/ChainAegis.md

```diff
@@ -0,0 +1,33 @@
+**About this app:**
+
+ChainAegis provide Rugpull alert, security incident alert, and you can check an address's reputation and risk types. Of course, you can also apply for other security services such as smart contract auditing, on-chain analysis, asset tracking, and data APIs.
+
+**To get started:**
+
+1. Install the ChainAegis Miniapp from the Collab.Land Marketplace
+
+2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.
+
+3. Your community members can check whether the address is risk through _/chainaegis check_, and they can also report addresses and risk details through _/chainaegis report_.
+
+4. Your community members can apply for other security services such as smart contract auditing, on-chain analysis, asset tracking, data apis through _/chainaegis request_
+
+**Commands**
+
+| **Command** | **Description** |
+| --- | --- |
+| /chainaegis alert | Get the latest 5 Rugpull and 5 security incidents. |
+| /chainaegis check | Check the address risk and labels on the Ethereum/BNB Chian/Bitcoin/Tron network. |
+| /chainaegis report | Report a risk address and its details to ChainAegis and add it to the blacklist. |
+| /chainaegis request audit | Apply for a smart contract audit for your project and we will contact you upon receipt. |
+| /chainaegis request on-chain-analysis | Apply for on-chain analysis service and we will contact you upon receipt. |
+| /chainaegis request kyt | Apply for ChainAegis KYT and we will contact you upon receipt. |
+| /chainaegis request fund-track | Apply for ChainAegis fund track service and we will contact you upon receipt. |
+| /chainaegis request data-api | Apply for ChainAegis data APIs and we will contact you upon receipt. |
+
+**Support**
+
+- Twitter: https://twitter.com/ChainAegis
+- Email: sharkteam@payegis.com
+- Discord: https://discord.com/invite/jGH9xXCjDZ
```

Comment:
make sure this discord invite will not expire. default invites expire after 7 days.

Reply:
I've update new discord invite, thanks for reminder.